sql-select/actor-forty.sql

```diff
@@ -0,0 +1,3 @@
+select *
+from "actors"
+where "actorId" = '40';
```

Comment:
Only use single quotes around strings. Don't put them around numbers, booleans, etc.

Postgres is smart enough to convert it for you, but it's a bad practice that can lead to bugs.

Check all your SQL. I marked a couple of them, but they are used in many places where they aren't needed.